backend/app.ts

```diff
@@ -1,87 +1,87 @@
 
-const db = prisma;
-const app = express();
+const db = prisma
+const app = express()
 
-app.use(morgan("dev"));
-app.use(cors());
-app.use(express.json());
+app.use(morgan('dev'))
+app.use(cors())
+app.use(express.json())
 
-const secretcode = process.env.SESSION_SECRET as string;
-const clientID = process.env.GOOGLE_CLIENT_ID as string;
-const clientSecret = process.env.GOOGLE_CLIENT_SECRET as string;
+const secretcode = process.env.SESSION_SECRET as string
+const clientID = process.env.GOOGLE_CLIENT_ID as string
+const clientSecret = process.env.GOOGLE_CLIENT_SECRET as string
 
 app.use(session({
-    secret: secretcode,
-    resave: true,
-    saveUninitialized: false,
-}));
+  secret: secretcode,
+  resave: true,
+  saveUninitialized: false
+}))
 app.use(passport.initialize())
 app.use(passport.session())
 
 passport.serializeUser((user: any, done: any) => {
-    return done(null, user.id)
+  return done(null, user.id)
 })
 
 passport.deserializeUser(async (id: string, done: any) => {
-    const user = await db.user.findFirst({
-        where: {
-            id: id
-        }
-    })
-    console.log('found user from cookie')
-    return done(null, user)
+  const user = await db.user.findFirst({
+    where: {
+      id
+    }
+  })
+  console.log('found user from cookie')
+  return done(null, user)
 })
 
 passport.use(new GoogleStrategy({
-    clientID: clientID,
-    clientSecret: clientSecret,
-    callbackURL: "/auth/google/callback"
+  clientID,
+  clientSecret,
+  callbackURL: '/auth/google/callback'
 },
-    async function verify(accessToken: any, refreshToken: any, profile: any, cb: any) {
-        // Called On successful authentication
-        // //find a user that has a matching google ID with the incoming profile ID
-        try {
-            const user = await db.user.findUnique({
-                where: {
-                    googleID: profile.id
-                }
-            })
+async function verify (accessToken: any, refreshToken: any, profile: any, cb: any) {
+  // Called On successful authentication
+  // //find a user that has a matching google ID with the incoming profile ID
+  try {
+    const user = await db.user.findUnique({
+      where: {
+        googleID: profile.id
+      }
+    })
 
-            if (!user) { // if user doesn't exist
-                // create a new user and store in database
-                const newUser = await db.user.create({
-                    data: {
-                        firstName: profile._json.given_name,
-                        lastName: profile._json.family_name,
-                        googleID: profile.id,
-                        email: profile.emails[0].value
-                    }
-                })
-                // return user object
-                cb(null, newUser)
-            } else {
-                cb(null, user)
-            }
-        } catch (error) {
-            cb(error, null)
+    if (user == null) { // if user doesn't exist
```

Comment:
hm i think we might want to ensure that the linter is enforcing triple === sign rather than double. not the end of the world if you can't figure it out, just add a TODO and we can investigate later